locales/en-US/app.ftl

```diff
@@ -413,6 +413,10 @@ MenuButtons--metaInfo--attempting-resymbolicate = Attempting to re-symbolicate p
 MenuButtons--metaInfo--currently-symbolicating = Currently symbolicating profile
 MenuButtons--metaInfo--cpu = CPU:
 MenuButtons--metaInfo--main-memory = Main memory:
+MenuButtons--moreInfo-button = More
+MenuButtons--moreInfo-title = Additional Profile Information
+MenuButtons--index--show-moreInfo-button = Show More
+MenuButtons--index--hide-moreInfo-button = Show Less
```

Comment:
nit: we have space here, so we can use more words. What about `Show more information` and `Show less information`?